tests/components/ps4/test_config_flow.py

```diff
@@ -38,6 +50,10 @@
     flow = ps4.PlayStation4FlowHandler()
     flow.hass = hass
 
+    # Init config manager.
+    manager = config_entries.ConfigEntries(hass, {})
+    hass.config_entries = manager
```

Comment:
Why do we need to replace the `config_entries` instance that is already set as an attribute on `hass` during `hass` test set up phase?

Reply:
@MartinHjelmare Because test fails if I dont. I used that excerpt from the tests for config entries.

`
@pytest.fixture

def manager(hass):

    """Fixture of a loaded config manager."""

    manager = config_entries.ConfigEntries(hass, {})

    manager._entries = []

    manager._store._async_ensure_stop_listener = lambda: None

    hass.config_entries = manager

    return manager

`

Reply:
Try to use `hass.config_entries` instead of a new `manager` instance.